### PR TITLE
com_search on back-end: Results links opens in backend window instead of new Tab

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #11852  .

### Summary of Changes
In the list of search results the link to the search now opens in a new window
An icon is added to visually indicate that it is opening in a new window and a screen reader only text link is also present.

### Testing Instructions
Enable search results in the admin search component
Perform a few searches in the front end
Return to the search component in the admin and you will see the log of the searched terms.
The number at the end of the row is the number of times that the term has been searched for.
Click on the link and it will open in a new window/tab

### CODE REVIEW

**Comment by zero-24** [comment]
Hmm look like you have mixed commits here?

**Comment by brianteeman** [comment]
@zero-24 ooops - quick and dirty fix to the PR - should be ok now

**Comment by zero-24** [comment]
Do you want to hard code the english text?

**Comment by brianteeman** [comment]
No - oops

File: administrator/components/com_search/views/searches/tmpl/default.php [hunk 1]
```diff
@@ -68,8 +68,8 @@
 					<?php if ($this->state->get('show_results')) : ?>
 					
 					<td class="center btns">
-						<a class="badge <?php if ($item->returns > 0) echo "badge-success"; ?>" href="<?php echo JUri::root(); ?>index.php?option=com_search&amp;view=search&amp;searchword=<?php echo JFilterOutput::stringURLSafe($item->search_term); ?>">
-							<?php echo $item->returns; ?></a>
+						<a class="badge <?php if ($item->returns > 0) echo "badge-success"; ?>" target="_blank" href="<?php echo JUri::root(); ?>index.php?option=com_search&amp;view=search&amp;searchword=<?php echo JFilterOutput::stringURLSafe($item->search_term); ?>">
+							<?php echo $item->returns; ?><span class="icon-out-2"><span class="element-invisible"><?php echo JText::_('JBROWSERTARGET_NEW'); ?></span></span></a>
```
Comment (andrepereiradasilva) [inline administrator/components/com_search/views/searches/tmpl/default.php]: please don't use two spans . i think you could use just
```html
<span class="icon-out-2 element-invisible"><?php echo JText::_('JBROWSERTARGET_NEW'); ?></span>
```

**Comment by brianteeman** [comment]
Ok I will check. That's how we do it elsewhere - nut doesn't mean it's
correct :)

File: administrator/components/com_search/views/searches/tmpl/default.php (hunk 1 above)
Comment (brianteeman) [inline administrator/components/com_search/views/searches/tmpl/default.php]: That doesnt work

**Comment by Martijn-Boomsma** [comment]
I have tested this item :white_check_mark: successfully on 0b789187fd1d87ca99d6de838dd397afaf6251e1<br /><br />1. Enabled search results in the admin search component
2. Performed a few searches in the front end
3. Returned to the search component in the admin and saw the log of the searched terms.
4. **Att.** The number at the end of the row is NOT the number of times that the term has been searched for (That is HITS) but the number of search result.
5. After applying the test I clicked on the link (Under results) and it indeed opened in a new window/tab on the Frontend (instead of same backend window).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12626">issues.joomla.org/tracker/joomla-cms/12626</a>.</sub>

**Comment by euismod2336** [comment]
I have tested this item :white_check_mark: successfully on 0b789187fd1d87ca99d6de838dd397afaf6251e1<br /><br />Link does indeed open in a new window, showing the search results.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12626">issues.joomla.org/tracker/joomla-cms/12626</a>.</sub>

**Comment by zero-24** [comment]
@brianteeman can we RTC or do you want to change something here?

**Comment by brianteeman** [comment]
Its fine for merging from my perspective. The suggestion from Andre did not
work.

**Comment by zero-24** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12626">issues.joomla.org/tracker/joomla-cms/12626</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**